api/src/main/java/io/strimzi/api/kafka/model/Kafka.java

```diff
@@ -109,7 +109,7 @@
     public static final List<String> RESOURCE_SHORTNAMES = unmodifiableList(singletonList(SHORT_NAME));
 
     private String apiVersion;
-    private ObjectMeta metadata;
+    private ObjectMeta metadata; // leave it for the generator / builder
```

Comment:
What is the point of the comment? I think it should be either removed or made more clear what does it mean.